kino-nextjs/app/components/navigation/SearchBar/SearchInput.tsx

```diff
@@ -19,9 +19,17 @@ export default function SearchMovies() {
   }, 600);
   return (
     <>
+      <label
+        htmlFor='searchMovies'
+        className='flex flex-row gap-1 text-custom_yellow self-center md:self-start'
+      >
+        Sök på din Bihjofilm!
```

Comment:
Nice job! Maybe change this to something like "Vilken film vill du se?" or something else so its not the same as the placeholder? :)

Reply:
Absolut 😀